src/styles/components/typeahead.less

```diff
@@ -0,0 +1,103 @@
+.form-control.form-control-group {
+
+  .typeahead {
+
+     .form-control {
+
+       &.bootstrap-typeahead-input-hint {
+
+        margin-bottom: 0 !important;
+        margin-top: 0 !important;
```

Comment:
Do we need these `!important` tags? How specific is the margin that we're overriding?